src/geo/map.js

```diff
@@ -72,46 +72,10 @@ var Map = Model.extend({
       }
     }.bind(this), this.RELOAD_DEBOUNCE_TIME);
 
-    this._initBinds();
-  },
-
-  _initBinds: function () {
-    this.layers.bind('reset', this._onLayersResetted, this);
-    this.layers.bind('add', this._onLayerAdded, this);
-    this.layers.bind('remove', this._onLayerRemoved, this);
+    this.layers.bind('reset', this._updateAttributions, this);
```

Comment:
The thing is that layers are actually reset before the map is instantiated... and attributions need to be updated. I'll think about a better approach right after we fix this.